### DISCUSSION TITLE
VTM rendering on Windows version missing

### DISCUSSION THREAD

**Discussion by Ebe66** [devemux86/cruiser#575] [Q&A] [answered]
The PC version of Cruiser is used by many people to make and test VTM rendering themes for the Wahoo line of bicycle computers.

It seems as if the option to switch to VTM rendering is no longer available in the 4.0.2. version of Cruiser. 
Is this correct or am I missing something?

When I try to open a VTM theme I now get a "unknown element style-line" error which in previous versions of Cruiser indicated that you needed to switch to the VTM rendering engine.

**Comment by devemux86** [comment, marked as answer]
Desktop Cruiser 4 has now hidden the VTM engine. You can still run it with:
```console
java -jar cruiser.jar vtm
```

OpenGL has various [issues](https://github.com/mapsforge/vtm/issues) & [reports](https://github.com/mapsforge/vtm/discussions/1128), but not many are interested to help.

In the future can use the VTM desktop application, see instructions [here](https://github.com/mapsforge/vtm/blob/master/docs/desktop.md).

---

[Cruiser 5](https://github.com/devemux86/cruiser/discussions/559) will only use the new Mapsforge engine.

**Comment by Ebe66** [comment]
Thanks @devemux86 for the quick answer, and for leaving in vtm rendering atm.
It works, so I can make some people happy!

When Cruiser 5 comes out could you please consider to keep a vtm capable release on here too?
The VTM desktop application you refer to does not provide a compiled version.

**Reply by devemux86** [reply]
Desktop Cruiser 4 releases (will) also contain the VTM engine.

So you can keep this verion if you want to use VTM on desktop.
